service/src/main/java/com/fecklessweasel/service/DepartmentUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.CodeContract;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.ServiceStatus;
+import com.fecklessweasel.service.objectmodel.University;
+import com.fecklessweasel.service.objectmodel.Department;
+
+/**
+ * Utility class for looking up universities from servlet request
+ * @author Anjana Rao
+ */
+public final class DepartmentUtil{
+    /**
```

Comment:
Javadoc comment goes on the function, not on class fields. If we build the Javadoc for this class it will say:

did: Private constructor to prevent instantiation.

DepartmentUtil's documentation will be blank.